src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_AnyAttribute.cs

```diff
@@ -0,0 +1,202 @@
+﻿using Xunit;
+using Xunit.Abstractions;
+using System.IO;
+using System.Xml.Schema;
+
+namespace System.Xml.Tests
+{
+    public class TC_SchemaSet_AnyAttribute : TC_SchemaSetBase
+    {
+        private ITestOutputHelper _output;
+        public bool bWarningCallback;
+        public bool bErrorCallback;
+        public int errorCount;
+        public int warningCount;
+        public bool WarningInnerExceptionSet = false;
+        public bool ErrorInnerExceptionSet = false;
+
+        public TC_SchemaSet_AnyAttribute(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        public void Initialize()
+        {
+            bWarningCallback = bErrorCallback = false;
+            errorCount = warningCount = 0;
+            WarningInnerExceptionSet = ErrorInnerExceptionSet = false;
+        }
```

Comment:
As i wrote earlier, i wanted to use project code style ( there are similar variables initializers in other classes). So should i change it or leave it like in other places?